docs/src/pages/basics/exporting-storybook/index.md

```diff
@@ -37,22 +37,10 @@ Additionally, you can deploy Storybook directly into GitHub pages with our [stor
 
 Or, you can export your storybook into the docs directory and use it as the root for GitHub pages. Have a look at [this guide](https://github.com/blog/2233-publish-your-project-documentation-with-github-pages) for more information.
 
-## Deploying to Vercel Now
+## Deploying to Vercel
 
-[Vercel Now](https://vercel.com/home) is a cloud platform for websites and serverless APIs, that you can use to deploy your Storybook projects to your personal domain (or a free `.now.sh` suffixed URL).
+[Vercel](https://vercel.com/home) is a cloud platform for hosting static sites and Serverless Functions, that you can use to deploy your Storybook projects to your personal domain (or a free `.now.sh` suffixed URL).
 
-- Install the [Vercel Now CLI](https://github.com/zeit/now):
+To deploy your Storybook project to Vercel, all that's required is to connect your [Git repository](https://vercel.com/docs/v2/git-integrations) and import the project. The build command, project directory, and project type will be automatically detected upon import.
```

Comment:
@mcsdevv It automatically detects `build-storybook`? what happens, for example, if i add storybook to a CRA app so there's both a `build` for the CRA app and a `build-storybook` also? is there any way to force it to deploy the storybook?

Reply:
It'll detect using dependencies and the build script. If you're deploying both at the same time, CRA will be chosen.

Reply:
Can you force it to show Storybook if that's what you want? Seems to me that's what the deleted instructions are showing...

Reply:
Absolutely, I'm just trying to understand the use case. It seems odd that you would have CRA and Storybook and only want to deploy Storybook. In my mind, you're more likely to want to deploy CRA or perhaps both - ideally both but as separate deployments which is coming soon!

For now, I'll add details about the build script to make it clearer, we can always adjust this later.

Reply:
Thanks for your patience. I wasn't trying to force back that documentation -- was just trying to understand the current state of things, since the "before" made sense to me and the "after" seemed opaque to me. Anyway will merge and am happy to update etc. any time!

Reply:
Totally understand! It's no good if the documentation gets worse, thanks for your thoughts - no doubt I'll be back in a couple of weeks as the final changes are implemented.